docs/user_guide/datasets/acs_income.rst

```diff
@@ -160,12 +162,7 @@ classification task.
 
 .. topic:: References:
 
-  .. [1] Frances Ding, Moritz Hardt, John Miller, Ludwig Schmidt `"Retiring Adult: New Datasets for Fair Machine Learning" <https://arxiv.org/pdf/2108.04884.pdf>`_,
-      Advances in Neural Information Processing Systems 34, 2021.
-
-  .. [2] R. Kohavi and B. Becker, UCI Machine Learning Repository: Adult Data Set, 01-May-1996. 
-      Available Online `[link] <https://archive.ics.uci.edu/ml/datasets/adult>`_.
-
-  .. [3] `"Group Quarters and Residence Rules for Poverty" <https://www.census.gov/topics/income-poverty/poverty/guidance/group-quarters.html>`_,
+  .. [CENSUS] `"Group Quarters and Residence Rules for Poverty" <https://www.census.gov/topics/income-poverty/poverty/guidance/group-quarters.html>`_,
```

Comment:
Can this be turned into BibTeX?

Reply:
Couldn't find a it's corresponding bibtex format, and I also don't anticipate it being cited anywhere else on our website.

Reply:
https://bibtex.eu/types/misc/

I'd really prefer for _everything_ to be in BibTeX.

Reply:
I can certainly try and make up my own bibtex entries 😆

Reply:
From looking at the results, it does look cleaner (obviously...) to have just 1 type of references, not two, so that's another argument in favor of what you're saying.

Reply:
Done!